packages/react-core/src/helpers/util.ts

```diff
@@ -333,19 +368,19 @@ export const getBreakpoint = (width: number): 'default' | 'sm' | 'md' | 'lg' | '
   if (width === null) {
     return null;
   }
-  if (width >= 1450) {
+  if (width >= globalWidthBreakpoints['2xl']) {
```

Comment:
super cool 😎

Reply:
Agreed! Thanks for this great suggestion! 🙌